modules/file-system/filestore/variables.tf

```diff
@@ -90,15 +90,10 @@ variable "filestore_tier" {
     condition = contains([
       "BASIC_HDD",
       "BASIC_SSD",
-      "HIGH_SCALE_SSD",
```

Comment:
Please keep HIGH_SCALE_SSD and ENTERPRISE in the `contains` list argument, but leave the error_message as it is. Logic: make the new recommendations obvious, but allow current legacy users to function when they have made an explicit choice.

Leave updated values in blueprint alone, too.

Create a follow-up task to add a `check` block that warns against use of legacy tiers ENTERPRISE and HIGH_SCALE_SSD.